templates/webTemplateFiles.yaml

```diff
@@ -124,3 +124,45 @@
   email: stefan@endrullis.de
   website: https://github.com/xylo/intellij-postfix-templates/wiki/PHP-web-templates:-xylo.misc
   description: Provides templates for miscellaneous use cases
+- lang: go
+  id: xylo.numbers
+  url: https://raw.githubusercontent.com/xylo/intellij-postfix-templates/master/templates/go/numbers.postfixTemplates
+  author: Stefan Endrullis
+  email: stefan@endrullis.de
```

Comment:
These web template files are your work.  Can you replace the author name and email by your name and email?